src/test/java/com/bobocode/hoverla/bring/context/BeanInitializerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.bobocode.hoverla.bring.context;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.mockito.Mockito.*;
```

Comment:
Let's avoid usage of * to be more explicit

Reply:
@DanielChanger I doubt it makes sense. Do you think `mock()` is not explicit enough? Mockito is the first framework you think of, when it comes to mocks, spies etc. Apart from that, I don't think that it is worth checking whether each and every PR doesn`t have star imports

Reply:
I say that you should use static imports without * 
`import static org.mockito.Mockito.mock;`

I'm not saying you have to use `Mockito.mock()` in your code

You'll still be able to use just `mock()` without class

By removing * we're just adding explicitness to imports, it's considered to be a good practice

Reply:
What if we have 5+ static imports of the same class? Or 10+? Should we also avoid star imports?

Reply:
it depends. In my practice only Sonar could prevent from a big amount of imports. 
But often it just a sign that you have to split your class. 
So, in this case I see no reason not to follow commonly accepted good practice.